packages/viewer/src/lib/scenes/colorMaterial.ts

```diff
@@ -13,8 +13,8 @@ export interface ColorMaterial {
 }
 
 const defaultColor: ColorMaterial = {
-  opacity: 100,
-  glossiness: 10,
+  opacity: 255,
+  glossiness: 50,
```

Comment:
Can we pull the default glossiness out as a constant to ensure it is the same everywhere it is used?

Reply:
exported the defaultColor referenced the value from there.